pkg/interface/src/apps/chat/components/lib/ship-search.js

```diff
@@ -119,35 +118,30 @@ export class ShipSearch extends Component {
 
       return (
         hay.startsWith(needle) ||
-        _.some(_.words(hay), s => s.startsWith(needle))
+        hay.split(/\s/).some(s => s.startsWith(needle))
```

Comment:
this is not technically equivalent, but it's close enough. Leaves in punctuation and numbers.